src/core/app_backend.tf

```diff
@@ -310,6 +318,12 @@ data "azurerm_key_vault_secret" "app_backend_PECSERVER_ARUBA_TOKEN_SECRET" {
   name         = "appbackend-PECSERVER-ARUBA-TOKEN-SECRET"
   key_vault_id = data.azurerm_key_vault.common.id
 }
+
+
+data "azurerm_key_vault_secret" "app_backend_APP_MESSAGES_API_KEY" {
+  name         = "appbackend-APP-MESSAGES-TOKEN-SECRET"
```

Comment:
also name :)

Reply:
I mean that also name must be aligned. something like `appbackend-APP-MESSAGES-API-KEY`

Reply:
Done in [02e984c](https://github.com/pagopa/io-infra/pull/142/commits/02e984c16c7f5ab218947c394a40c6ac899b982c)